aio/src/app/custom-elements/contributor/contributor.component.ts

```diff
@@ -15,15 +15,18 @@ import { CONTENT_URL_PREFIX } from 'app/documents/document.service';
             <div class="contributor-image"
                  [style.background-image]="'url('+pictureBase+(person.picture || noPicture)+')'">
                  <div class="contributor-info">
-                     <button *ngIf="person.bio" mat-button class="info-item">
+                     <button *ngIf="person.bio" mat-button class="info-item"
+                        [attr.aria-label]="'View bio of ' + person.name">
```

Comment:
OOC, could we also use "attribute binding" (which is slightly more concise):

```suggestion
                        aria-label="View bio of {{ person.name }}">
```

Reply:
the simple syntax without `attr.` seems to be allowed only for static content,
as you can see here:
![Screenshot at 2022-06-10 13-49-27](https://user-images.githubusercontent.com/61631103/173067801-ea68acc3-afea-4c78-a2b9-1c14fa6ef189.png)

 so I applied the string interpolation but had to keep the `attr.`